src/components/tabs/tabs.component.js

```diff
@@ -69,17 +69,11 @@ const Tabs = ({
   const [tabsInfos, setTabsInfos] = useState({});
 
   useLayoutEffect(() => {
-    const selectedTab =
-      selectedTabId || Children.toArray(children)[0].props.tabId;
-
-    if (!tabIds.includes(selectedTabId)) {
-      setTabStopId(React.Children.toArray(children)[0].props.tabId);
-    } else {
-      setTabStopId(selectedTab);
-    }
+    const selectedTab = selectedTabId || filteredChildren[0].props.tabId;
 
     setSelectedTabIdState(selectedTab);
-  }, [children, selectedTabId, tabIds]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Could we add `selectedTab`, `selectedTabId` and `filteredChildren` to the dependency array here?

Reply:
We only want this to happen onMount rather than every-time a prop or value updates, I could move it to where define the state and setter but it caused coverage issues so went with this for now

Reply:
Thinking about a little more, I think you make a good point I'll refactor it initialise the sate with a value rather than rely on the hook